utils/data_version_control.py

```diff
@@ -5,7 +5,7 @@
 
 import git
 from config import DATA_DIR, PROJECT_ROOT_DIR
-from pygit2 import Repository  # type: ignore
+from pygit2 import Repository
```

Comment:
Yes, this is because of the `ignore_missing_imports` is moved in the `pyproject.toml` now, we should not need it before but `ignore_missing_imports` was wrongly placed.